articles/how-to-use-kanban-to-manage-your-next-project/index.md

```diff
@@ -0,0 +1,132 @@
+
+- [Trello](https://trello.com/en)
+- [Jira](https://www.atlassian.com/software/jira)
+- [Kanbanize](https://kanbanize.com/)
+- [Kanban Tool](https://kanbantool.com/)
+- [Hygger](https://hygger.io/)
+
+### How to apply Kanban in a project
+
+It is straightforward to adopt Kanban in your next project. The most crucial requirement is to get your team to understand what Kanban is. Make sure you understand the Kanban principles.
+
+#### Kanban principles
+
+- **Start with what you have.** Changes to the current workflow and process can be introduced incrementally. You don't have to restructure your team either. 
+- **Pursue continuous improvement.** Minimal incremental changes are encouraged to avoid resistance in the team.
+- **Embrace leadership at all levels.** Kanban encourages everyone to be a leader by making continuous improvements at a personal level. The team should be self-managing.
+- **Follow the current process and roles.** You do not have to create new roles or processes. The responsibilities and titles in your team remain the same.
+
+**Step 1:** Get a Kanban board. Choose between a physical board or a digital board. Discuss with your team and decide what fits you best.
+**Step 2:** Identify the different states in your existing process and put the workflow on the Kanban board.
+**Step 3:** Identify the commitment point and the delivery point.
+**Step 4:** Set the Work in Progress Limits.
+**Step 5:** Identify the work items. Identify the different tasks and put them on cards.
+**Step 6:** Place cards in the appropriate stages of the workflow.
+**Step 7:** Work on the cards. 
+**Step 8:** Identify the bottlenecks.
+**Step 9:** Work on the bottlenecks and improve continuously.
+
+### Kanban vs. Scrum
+
+Both Scrum and Kanban are frameworks for agile project management. You can learn more about Scrum in the article [Introduction to Scrum](https://www.section.io/engineering-education/introduction-to-scrum/). The table below shows the core differences between Scrum and Kanban.
+
+#### Differences between Scrum and Kanban 
+
+Scrum |  Kanban
+---|---
+In Scrum, every person involved has a role and responsibilities.| There are no fixed roles and responsibilities.
+Scrum is based on durations. For example, a sprint duration must be 2 to 4 weeks| Kanban is not based on durations. There is a continuous flow of the development process.
+Scrum is best for projects with changing requirements. | Kanban is best for projects with minimal requirements changes.
+Scrum focuses on time management and better utilization of resources. | Focuses on continuous improvement to higher productivity and more efficiency.
+The Scrum Master leads the scrum team. | Leadership is encouraged at all levels.
+
```

Comment:
Great comparison table!
It would be good if you could also add an example of a project where Scrum would be more suitable, and also a project where Kanban would be more suitable.